### PR TITLE
Fix ts prerequisistes

### PR DESCRIPTION
**Pull request by abcdefg30**
As wished by @Phrohdoh . :)

Removed the `Owner:` trait and added `Prerequisistes:`.

### CODE REVIEW

**Comment by obrakmann** [comment]
Will close #2187

**Comment by pchote** [comment]
I believe that this is the last blocker for removing the legacy Owner code, too.

File: mods/ts/rules/structures.yaml [hunk 1]
```diff
@@ -405,8 +404,7 @@ GASAND:
 	Buildable:
 		Queue: Defense
 		BuildPaletteOrder: 1000
-		Prerequisites: proc
-		Owner: gdi
+		Prerequisites: ~structures.gdi
```
Comment (Mailaender) [inline mods/ts/rules/structures.yaml]: `proc, ~structures.gdi`

File: mods/ts/rules/structures.yaml [hunk 2]
```diff
@@ -433,8 +431,7 @@ GAWALL:
 	Buildable:
 		Queue: Defense
 		BuildPaletteOrder: 1001
-		Prerequisites: gapile
-		Owner: gdi
+		Prerequisites: ~structures.gdi
```
Comment (Mailaender) [inline mods/ts/rules/structures.yaml]: `gapile, ~structures.gdi`

File: mods/ts/rules/structures.yaml (hunk 1 above)
Comment (Mailaender) [inline mods/ts/rules/structures.yaml]: Nevermind, that legacy pre-requisite probably makes no sense.

File: mods/ts/rules/structures.yaml [hunk 3]
```diff
@@ -1256,8 +1238,7 @@ NAPULS:
 	Buildable:
 		Queue: Defense
 		BuildPaletteOrder: 90
-		Prerequisites: radar
-		Owner: nod, gdi
+		Prerequisites: radar, ~structures.gdi
```
Comment (Mailaender) [inline mods/ts/rules/structures.yaml]: http://cnc.wikia.com/wiki/Electromagnetic_Pulse_cannon is used by every faction.

**Comment by abcdefg30** [comment]
Updated.

**Comment by Mailaender** [comment]
:+1:

**Comment by Mailaender** [comment]
Looks like I have to rebase https://github.com/OpenRA/OpenRA/pull/7191 which has been ignored so far once this is merged.

**Comment by LipkeGu** [comment]
i already worked on these Owner Removals ...

**Comment by Mailaender** [comment]
Nobody knew. Be sure to communicate it in IRC or GitHub next time.

**Comment by LipkeGu** [comment]
i had

**Comment by LipkeGu** [comment]
```
[01:20:31]	<LipkeGu> yes
[01:20:42]	<pchote> "RequireOwner: false"
[01:20:48]	<pchote> on ClassicProductionQueue@Building
[01:21:19]	<pchote> afk again
[01:21:27]	<LipkeGu> i copied the player.yaml from ra
[01:23:11]	<LipkeGu> it works now :)
[01:26:37]	*** Clemb has quit IRC (Quit: night)
[01:28:15]	*** m1cr0man has quit IRC (Read error: Connection reset by peer)
[01:39:06]	<LipkeGu> ProvidesCustomPrerequisite:
[01:39:06]	<LipkeGu> Race: soviets
[01:39:06]	<LipkeGu> Prerequisite: barracks.soviets
[01:39:13]	<LipkeGu> works also :)
[01:39:44]	<LipkeGu> it should be able now to implement this in TS :)
[01:57:27]	<LipkeGu> ok i have now understand this logic and should now be able to port TS to this scheme :) 
```

hour later...

```
[12:05:42]	<orabot> Youtube: Awesome Games Done Quick 2015 - Part 7 - TASbot plays a mystery game by dwangoAC and p4plus2
[12:05:45]	<Scott`> and is newer
[12:10:16]	*** DreadKnight has joined #openra
[12:12:15]	<LipkeGu> scott i had nearby ported the entire rules to the new scheme
[12:12:37]	<LipkeGu> in TS
[12:15:48]	*** jack_____ has joined #openra 
```

**Comment by Mailaender** [comment]
That is unfortunate.

**Comment by Mailaender** [comment]
You are already a member of https://github.com/orgs/OpenRA/teams/bug-assignees though.

**Comment by LipkeGu** [comment]
yes but i didnt have rights for this =(

**Comment by Phrohdoh** [comment]
:+1: thanks @abcdefg30!

**Pull request merged**